sitemap.functions.php

```diff
@@ -367,7 +371,7 @@ function scan_url($url)
         return $depth--;
     }
 
-    if ($noindex && preg_match('/content="noindex"/', $html)) {
+    if ($noindex && (preg_match_all('/\<meta.*?\>/mis',$html,$ar) and strstr(join(',',$ar[0]),'noindex'))) {
```

Comment:
This is a really interesting way of approaching the problem. Has room for improvement, but I'm a fan of it.